.github/actions/github-release/action.yml

```diff
@@ -0,0 +1,32 @@
+name: github release
+description: Creates a GitHub release for the shared packages.
+inputs:
+  gitHubToken:
+    description: Personal Access Token (PAT) for GitHub used to create the release
+    required: true
+  packageFolder:
+    description: The folder for generated NuGet packages
+    required: true
+  tag:
+    description: The Git tag to create and associate with the release
+    required: true
+  title:
+    description: The title for the GitHub release
+    required: true
+
+runs:
+  using: composite
+  steps:
+  - name: Release
+    shell: bash
+    run: |
+      gh release create "${{ inputs.tag }}" \
```

Comment:
Unsure if I need to tag the commit first. I'd imagine I have to... so I'll look how to best do it through our automation. I've done similar things before like [this](https://github.com/wsugarman/durabletask-azurestorage-scaler/blob/main/.github/actions/github-release/scripts/CreateRelease.mjs).

Reply:
The `gh release create` command will automatically create a tag if necessary. I've passed the commit SHA as well to ensure it correctly tags the proper code. See https://cli.github.com/manual/gh_release_create